netstate/auth/process_api_key.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+package main
+
+import  (
+	"fmt"
+	"os"
+	"flag"
+)
+ 
+type httpHeaders struct {
+	XApiKey string
+	Connection string 
+	CacheControl string
+	UpgradeInsecureRequests string 
+	Accept string
+	AcceptLanguage string
+}
+
+func main() {
```

Comment:
we've been putting test main methods inside storj/examples/<program>/main.go.

that way you can have a program that uses your library, but it's more clear it's not part of the main library

Reply:
great! didn't know that. will do.